stdlib/public/CompatibilityOverride/CompatibilityOverrideConcurrency.def

```diff
@@ -282,6 +282,9 @@ OVERRIDE_TASK_GROUP(taskGroup_addPending, bool,
                     swift::, (TaskGroup *group, bool unconditionally),
                     (group, unconditionally))
 
+OVERRIDE_TASK_GROUP(task_cancel_group_child_tasks, void,
```

Comment:
See earlier comment — this has moved, which the comment at the top of this file suggests is an ABI break.